stories/design/get-started.stories.mdx

```diff
@@ -4,14 +4,37 @@ import { Meta } from "@storybook/addon-docs";
 
 # Getting started as a designer
 
-Styles are documented in the [Compound Styles Figma document](https://www.figma.com/file/PpKepmHKGikp33Ql7iivbn/Compound-Styles?node-id=22%3A2763&t=UAtRgYRpQ2xofvM3-1)
+Available styles can be found in the [Compound Styles Figma document](https://www.figma.com/file/PpKepmHKGikp33Ql7iivbn/Compound-Styles?node-id=22%3A2763&t=UAtRgYRpQ2xofvM3-1)
+They are generated from our set of design tokens.
 
-Compound is built on top of design tokens, for convienience you can switch between
-themes using the [Themer Figma plugin](https://www.figma.com/community/plugin/731176732337510831/Themer)
+Themes can be switched using the [Themer Figma plugin](https://www.figma.com/community/plugin/731176732337510831/Themer)
 
-Follow these instructions to [setup themer and start using design tokens in your Figma documents](https://github.com/vector-im/compound-design-tokens/blob/develop/docs/sync-tokens.md#%EF%B8%8F-use-tokens-in-a-figma-document)
+In order to use this plugin, you will need to
 
-## Feedback
+- Get the JSONBin access token in your password manager
+- Input the bin URL: https://api.jsonbin.io/v3/b/6400f859c0e7653a05816945
```

Comment:
@gsouquet is it wise to publish the bin URL here? or should we just include that in the password manager?